README.md

```diff
@@ -185,7 +186,7 @@ it to actually send messages in development mode.
 
 ## Testing
 
-Testing is done in three ways; using rspec, rails test and feature tests.
+Testing is done in two ways; using rspec and via rails tests.
```

Comment:
Ahh, the feature tests are there, just undocumented for some reason.

```shell
bundle exec cucumber
```

Although given they are mostly legacy tests, I usually let the CI handle that bit for me.